doc/notebooks_multilabel_classification.rst

```diff
@@ -0,0 +1,10 @@
+Multi-label Classification notebooks
+===========================
+
+The following examples present advanced analyses
+on multi-label classification problems with different 
+methods proposed in MAPIE.
+
+1. Overview of Risk Control for Multi-Label Classification : 
+`notebook <https://github.com/scikit-learn-contrib/MAPIE/tree/master/notebooks/classification/tutorial_multilabel_classification.ipynb>`_
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
UPDATE: please add the same number of '-' characters as in the previous line (this causes display problems in the documentation).

Reply:
Done